Cargo.toml

```diff
@@ -55,7 +55,7 @@ nng = { version = "1.0", optional = true }
 [features]
 default = ["reqwest/default-tls"]
 gaggle = ["nng"]
-rustls = ["reqwest/rustls-tls"]
+rustls = ["reqwest/rustls-tls", "tokio-tungstenite/rustls-tls]
```

Comment:
You're missing an end quote here.

Reply:
Yeah, did this by hand in the GitHub editor to try and avoid creating a fork, but messed up :( Fixed now